Assignment/oddeven.txt

```diff
@@ -0,0 +1,21 @@
+/* [16] WAP to check number is Even or Odd
+ID : 23BCA261 */
+
+#include<iostream.h>
+#include<conio.h>
+void main()
+{
+	int n;
+	clrscr();
+	cout<<"Enter a number : ";
+	cin>>n;
+	if(n%2==0)

```

Comment:
can i write n/2==0?

Reply:
no we cant. because we have to check remainder and % modulas operator gives remainder. so we have to write n%2.

Reply:
correct.